integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImageOverride.java

```diff
@@ -0,0 +1,246 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Simple JUnit test file used for testing Model in Image.
+ *
+ * <p>This test is used for creating domain using model in image.
+ */
+
+public class ItModelInImageOverride extends MiiBaseTest {
+  private static Operator operator;
+  private static Domain domain;
+  private static String domainNS;
+  private static String testClassName;
+  private static StringBuffer namespaceList;
+  private static final String configMapSuffix = "-mii-config-map";
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception exception
+   */
+  @BeforeAll
+  public static void staticPrepare() throws Exception {
+    namespaceList = new StringBuffer();
+    testClassName = new Object() {
+    }.getClass().getEnclosingClass().getSimpleName();
+    // initialize test properties and create the directories
+    initialize(APP_PROPS_FILE, testClassName);
+  }
+
+  /**
+   * This method gets called before every test. It creates the result/pv root directories
+   * for the test. Creates the operator and domain if its not running.
+   *
+   * @throws Exception exception if result/pv/operator/domain creation fails
+   */
+  @BeforeEach
+  public void prepare() throws Exception {
+    createResultAndPvDirs(testClassName);
+
+    // create operator1
+    if (operator == null) {
+      Map<String, Object> operatorMap = createOperatorMap(getNewSuffixCount(),
+          true, testClassName);
+      operator = TestUtils.createOperator(operatorMap, RestCertType.SELF_SIGNED);
+      Assertions.assertNotNull(operator);
+      domainNS = ((ArrayList<String>) operatorMap.get("domainNamespaces")).get(0);
+      namespaceList.append((String)operatorMap.get("namespace"));
+      namespaceList.append(" ").append(domainNS);
+    }
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories.
+   *
+   * @throws Exception exception
+   */
+  @AfterAll
+  public static void staticUnPrepare() throws Exception {
+    tearDown(new Object() {
+    }.getClass().getEnclosingClass().getSimpleName(), namespaceList.toString());
+
+    LoggerHelper.getLocal().info("SUCCESS");
+  }
+
+  /**
+   * Create a domain using model in image and having configmap in the domain.yaml
+   * before deploying the domain. After deploying the domain crd,
+   * re-create the configmap with a model file that define a JDBC DataSource
+   * and update the domain crd to new config map and change domain
+   * restartVersion to reload the model, generate new config and initiate a
+   * rolling restart.
+   *
+   * @throws Exception exception
```

Comment:
why?

Reply:
I removed and change the doc to "to new config map" because cm name remains same.

   * Create a domain using model in image and having configmap in the domain.yaml
   * before deploying the domain. After deploying the domain crd,
   * re-create the configmap with a model file that define a JDBC DataSource
   * and update the domain crd to change domain restartVersion
   * to reload the model, generate new config and initiate a rolling restart.
 
The purpose of changing restartVersion is to trigger Inspector to do rolling restart per Tom and Johnny